homu/server.py

```diff
@@ -1,7 +1,8 @@
 import hmac
 import json
 import urllib.parse
-from .main import PullReqState, parse_commands, db_query, INTERRUPTED_BY_HOMU_RE, synchronize
+from .main import (PullReqState, parse_commands, db_query,
+                   INTERRUPTED_BY_HOMU_RE, synchronize)
```

Comment:
Thanks for the feedback. I'll try a unilateral switch to block-style indent to see what goes wrong -- I feel like we have a few spots where it'll be adding unduly many lines (like https://github.com/servo/homu/pull/103/files#diff-7a3e9e91f0de1349c2561ed96ae5e206R511) but on the whole it's probably worthwhile for consistency.